catalystwan/integration_tests/feature_profile/sdwan/topology/test_topology.py

```diff
@@ -0,0 +1,97 @@
+import os
+import unittest
+from ipaddress import IPv4Address
+from typing import cast
+from uuid import UUID
+
+from catalystwan.api.configuration_groups.parcel import as_global
+from catalystwan.api.feature_profile_api import ServiceFeatureProfileAPI, TopologyFeatureProfileAPI
+from catalystwan.models.configuration.feature_profile.sdwan.service.lan.vpn import LanVpnParcel
+from catalystwan.models.configuration.feature_profile.sdwan.topology.custom_control import CustomControlParcel
+from catalystwan.models.configuration.feature_profile.sdwan.topology.hubspoke import HubSpokeParcel
+from catalystwan.models.configuration.feature_profile.sdwan.topology.mesh import MeshParcel
+from catalystwan.session import ManagerSession, create_manager_session
+
+
+class TestTopologyFeatureProfile(unittest.TestCase):
```

Comment:
```suggestion
class TestTopologyFeatureProfile(TestFeatureProfileModels):
```

Can we inherit from `catalystwan/integration_tests/feature_profile/sdwan/base.py` ?

Reply:
sure let me check this